session_0/item.rb

```diff
@@ -41,5 +49,14 @@ def discounted?
   # Note: If there are no items for category, stock price for category
   # should be zero.
   def self.stock_price_by_category(items)
+    stock_prices = Hash.new
+    stock_prices = {1 => 0, 2 => 0, 3 => 0, 4 => 0}
+    i = 0
+    l = items.length
+    while i < l do
```

Comment:
since we know the size of categories before hand, it would be better to use something like
a for loop or .each construct instead
https://mixandgo.com/learn/how-to-use-ruby-each